packages/fetch-impl/src/dispatcher.ts

```diff
@@ -0,0 +1,18 @@
+import { Agent, Dispatcher } from 'undici';
+
+let cachedDispatcher: Agent;
+export function getDefaultDispatcher(): Dispatcher {
+    if (!cachedDispatcher) {
+        cachedDispatcher = new Agent({
+            // FIXME: https://github.com/nodejs/undici/issues/2808
+            // allowH2: true,
+            // See https://github.com/solana-labs/solana-web3.js/pull/2179 for
+            // strategies on how to tune the number of connections through benchmarking.
+            connections: 32,
```

Comment:
I'm starting to regret this in light of my latest benchmarks in #2179.

Reply:
If we're not creating a more performant Agent by providing these opinionated default values then that's good right? At least we don't need to be opinionated here.

Reply:
Yeah, I think I might tear this default `Agent` out until we have evidence that `undici` with _no_ dispatcher is insufficient. The `getLatestBlockhash` benchmarks would indicate that it does a fine job of scheduling requests without one.